x-pack/plugins/security_solution/public/timelines/components/side_panel/event_details/flyout/footer.tsx

```diff
@@ -43,6 +44,12 @@ interface AddExceptionModalWrapperData {
   ruleName: string;
 }
 
+const StyledFooter = styled(EuiFlyoutFooter)`
+  padding-inline: ${({ theme }) => theme.eui.euiSizeM};
+  padding-block: ${({ theme }) => theme.eui.euiSizeM};
+  background-color: transparent;
+`;
```

Comment:
We should also consider moving these reusable components to `/public/common/components`, this would let us know the changes may affect different places. The expandable flyout should not be using components/hooks from the Timeline.

Reply:
yes it shouldn't, and yes we should move these components out, but this is a much bigger effort and I believe should be done in a separate PR. It will be a nice cleanup though!

Reply:
`EuiFlyoutFooter` has certain built-in styles that only work when it is nested inside `EuiFlyout`. Since timeline is not nested in a `EuiFlyout` parent, those styles do not apply Because of this, timeline side panel footer looks like below:

![image](https://github.com/elastic/kibana/assets/7485038/88c995dc-453b-4d05-8bc1-94b36c851a05)


instead of looking like this:

![image](https://github.com/elastic/kibana/assets/7485038/bccabd50-1c68-4429-abd4-5f776e2e86de)

Infact, the whole, Timeline body ( table, and side panel ) styling is currently very messy becuase of which this patch work need to be applied.

This particular components is not reusable and is not use used anywere else so I think this part of styling is okay. And any other kind of refactoring should be taken separate PR.

Reply:
> This particular components is not reusable and is not use used anywere else so I think this part of styling is okay. And any other kind of refactoring should be taken separate PR.

while this specific `StyledFooter ` component isn't used anywhere, like I mentioned in you are using it within `FlyoutFooterComponent` which IS used in other places (expandable flyout), that's why I wanted to raise a point of making sure these other places were checked as part of the PR review. That's it 😄

Reply:
Hmm.. okay thanks for pointed that out. Let me double checks its impact on expandable flyout or anywhere it is used.

Reply:
@PhilippeOberti , Although, it does not impact Expandable flyout and expandable flyout applies it own style on this footer because current styles are just wrong.

But I understand your point and I have lifted the style customization one component above which specific to timeline.

Added here : 05d764c071a90d808c7e49f7a518add927c2e453